lib/+mwbs/@Contacts/Contacts.m

```diff
@@ -253,7 +253,7 @@
         % computes the Jacobian and J_dot_nu relative to the vertices (Not the sole frames!)
         [J_print, JDot_nu_print] = compute_J_and_JDot_nu_in_contact_frames(obj, robot, num_inContact_frames, num_vertices);
         
-        % computes the JTilde and JDOTTilde * nu fot the spilit points in the (possible) closed chains
+        % computes the JTilde and JDOTTildeThis function writes the local coordinates of the foot vertices in a cell array format * nu fot the spilit points in the (possible) closed chains
```

Comment:
Is this intended?

Reply:
No, sorry it was a mistake. It is corrected in ee731c270d7a7941606d1a98af328b5b4b6f6d52